src/main/java/de/retest/recheck/review/ignore/matcher/ElementClassMatcher.java

```diff
@@ -0,0 +1,52 @@
+package de.retest.recheck.review.ignore.matcher;
+
+import java.util.Objects;
+import java.util.regex.MatchResult;
+import java.util.regex.Pattern;
+
+import de.retest.recheck.review.ignore.io.RegexLoader;
+import de.retest.recheck.ui.descriptors.Element;
+
+public class ElementClassMatcher implements Matcher<Element> {
+
+	public static final String CLASS_KEY = "class";
+
+	private final String classValue;
+
+	public ElementClassMatcher( final Element element ) {
+		this( Objects.toString( element.getIdentifyingAttributes().get( CLASS_KEY ), "" ) );
+	}
+
+	private ElementClassMatcher( final String classValue ) {
+		this.classValue = classValue;
+	}
+
+	@Override
+	public boolean test( final Element element ) {
+		final String classValue = element.getIdentifyingAttributes().get( CLASS_KEY );
+		return classValue != null && classValue.matches( this.classValue );
+	}
+
+	@Override
+	public String toString() {
+		return String.format( ElementClassMatcherLoader.FORMAT, classValue );
+	}
+
+	public static final class ElementClassMatcherLoader extends RegexLoader<ElementClassMatcher> {
+
+		private static final String CLASS = CLASS_KEY + "=";
+
+		private static final String FORMAT = CLASS + "%s";
+		private static final Pattern REGEX = Pattern.compile( CLASS + "(.+)" );
```

Comment:
What do we do when one wants to address multiple classes, as one single class rarely fits?

```text
# with the above changes, this technically is not equal
matcher: class=flash-error
# do we want a separate syntax for that (i.e. contains)?
matcher: class~=flash 
# escape multiple with spaces, if so, when do we care about order?
matcher: class="flash-error flash"
matcher: class~="flash-error flash"
```

```
<div id="ajax-error-message" class="ajax-error-message flash flash-error"></div>
```

Reply:
Not relevant as it only appears once (see above)?

Reply:
Adapted the implementation accordingly, still open to discuss:

```
matcher: class=one two
# or
matcher: class="one two"
```

And one more thing: The `Set` currently doesn't guarantee order, so if the above is loaded, it may be saved as "two one"—which we have to handle.

Reply:
So I've adapted the things we have discussed, but I'm not sure if we should require the quoted variant over the unquoted one. Consider:

```
# should work
matcher: class=one
matcher: class="one"
matcher: class="one two"

# should not work
matcher: class=one two
```

This really complicates the implementation and tests, and there are several ways to do it. To come up with a proper solution, I'd suggest to further discuss and possibly pair this on Monday.